### ISSUE TITLE
Sort header not announcing "sorted by" status

### ISSUE THREAD

**Issue by jelbourn** [angular/components#11135]
### Summary
Upon Screen Reader focus set on column header for sort, does not announce 'sorted by' status.

### Reproduction on material.angular.io
1. Linear navigate to ' Components ' menu item and click 'Enter' to open components page.                                                                                                                                                                                                                                          
2. Linear navigate forward and set focus on expand collapse left hand menu item ' Navigation'                                                                                                                                                                                                                                  
3. Click 'Enter' key to expand OR collapse this menu item                                                                                                                                                                                                                                                                                       
4. Linear navigate into 'sort header' & click ENTER keystroke.                                                                                                                                                                                                                                                                                   
5. Navigate into tabs "Examples" and Click ENTER                                                                                                                                                                                                                                                                                 
6. Navigate and set focus on column 1, row 1 i.e. 'dessert (100g) ' and click 'ENTER' OR ' SPACEBAR' to sort                               Observe Screen Reader does not announce the status of the table , column being sorted by. Also does not announce the data has been sorted at all. No status is being announced.

**Comment by vick08** [comment]
Hi.
The issue is showing as has been fixed, however, it is still open. Has it been verified with screen readers?

**Comment by jelbourn** [comment]
The pull request to fix this was never merged due to failures inside Google. It looks like people are depending on the internal structure of the sort-header in their unit tests, so we'll need to invest some time to refactor those downstream tests first.

**Comment by jelbourn** [comment]
The pull request to fix this has been merged.
